python/ray/utils.py

```diff
@@ -561,6 +560,7 @@ def try_to_create_directory(directory_path, warn_if_exist=True):
             if e.errno != errno.EEXIST:
                 raise e
             if warn_if_exist:
+                logger = logging.getLogger("ray")
```

Comment:
is this change important?

Reply:
no. Since I use warn_if_exist = False. Just don't want to create unnecessary logger.